packages/web-pkg/src/components/QuotaSelect.vue

```diff
@@ -142,10 +137,10 @@ export default {
           value,
           displayValue,
           displayUnit: 'GB',
-          error: this.$gettextInterpolate(
-            this.$gettext('Max quota limit of %{ limit } GB exceeded'),
-            { limit: this.maxQuota * Math.pow(10, -9) }
-          ),
+          error: this.$gettext('Please enter a value less than %{ maxQuota } GB', {
```

Comment:
It should be `...less or equal than...`, no?

Reply:
yes, wrong in the ticket